cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,19 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+		Example: `# Create a Cluster Workflow Template with Default Options:
+		argo cluster-template create FILE1.yaml
+
+		# Create a Cluster Workflow Template in JSON Format:
```

Comment:
```suggestion
		# Create a Cluster Workflow Template and print it as JSON:
```

- it doesn't "create [...] in JSON", as the input is YAML, it prints it out as JSON
- simplify a bit, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- "Format" should not be capitalized, same as other comments

Reply:
Can you change these examples to use YAML instead? That is the most common format for k8s users, so an example matching that would be better